.github/workflows/build.yml

```diff
@@ -68,15 +68,20 @@ jobs:
             CI_TAG="ci"
           fi
           echo "::set-env name=TAG::${CI_TAG}"
-          echo "::set-env name=BRANCH::{BRANCH}"
+          echo "::set-env name=BRANCH::${BRANCH}"
           echo "BRANCH: ${BRANCH}"
           echo "TAG: ${CI_TAG}"
 
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+        with:
+          platforms: all
+
       - name: Set up Docker Buildx
         id: buildx
-        uses: crazy-max/ghaction-docker-buildx@v1
+        uses: docker/setup-buildx-action@v1
         with:
-          buildx-version: latest
+          version: latest
```

Comment:
Should we use a fixed version, instead of `latest` ?

Reply:
nvm, I think latest is fine, as we were using that earlier.

Reply:
The quickstart had the latest version and taking latest will also help get future fixes without changing anything in the files.